Author: <andrea.de.iacovo@gmail.com>
Description: This makes us able to change rss_language directly from the admin page. Thanks to Lionel Elie Mamane
--- a/wp-admin/options-reading.php
+++ b/wp-admin/options-reading.php
@@ -65,6 +65,10 @@ include('admin-header.php');
 <td><input name="posts_per_rss" type="text" id="posts_per_rss" value="<?php form_option('posts_per_rss'); ?>" class="small-text" /> <?php _e('posts') ?></td>
 </tr>
 <tr valign="top">
+<th scope="row"><?php _e('RSS feed language') ?></th>
+<td><input name="rss_language" type="text" id="rss_language" value="<?php form_option('rss_language'); ?>" size="2" /></td>
+</tr>
+<tr valign="top">
 <th scope="row"><?php _e('For each article in a feed, show') ?> </th>
 <td><fieldset><legend class="screen-reader-text"><span><?php _e('For each article in a feed, show') ?> </span></legend>
 <p><label><input name="rss_use_excerpt"  type="radio" value="0" <?php checked(0, get_option('rss_use_excerpt')); ?>	/> <?php _e('Full text') ?></label><br />
--- a/wp-admin/options.php
+++ b/wp-admin/options.php
@@ -27,7 +27,7 @@ $whitelist_options = array(
 	'misc' => array( 'use_linksupdate', 'uploads_use_yearmonth_folders', 'upload_path', 'upload_url_path' ),
 	'media' => array( 'thumbnail_size_w', 'thumbnail_size_h', 'thumbnail_crop', 'medium_size_w', 'medium_size_h', 'large_size_w', 'large_size_h', 'image_default_size', 'image_default_align', 'image_default_link_type', 'embed_autourls', 'embed_size_w', 'embed_size_h' ),
 	'privacy' => array( 'blog_public' ),
-	'reading' => array( 'posts_per_page', 'posts_per_rss', 'rss_use_excerpt', 'blog_charset', 'show_on_front', 'page_on_front', 'page_for_posts' ),
+	'reading' => array( 'posts_per_page', 'posts_per_rss', 'rss_use_excerpt', 'rss_language', 'blog_charset', 'show_on_front', 'page_on_front', 'page_for_posts' ),
 	'writing' => array( 'default_post_edit_rows', 'use_smilies', 'ping_sites', 'mailserver_url', 'mailserver_port', 'mailserver_login', 'mailserver_pass', 'default_category', 'default_email_category', 'use_balanceTags', 'default_link_category', 'enable_app', 'enable_xmlrpc' ),
 	'options' => array( '' ) );
 if ( !defined( 'WP_SITEURL' ) ) $whitelist_options['general'][] = 'siteurl';
